packages/react-native-bpk-component-icon/src/BpkIcon.js

```diff
@@ -22,18 +22,22 @@ import PropTypes from 'prop-types';
 import {
   spacingBase,
   spacingLg,
-  colorSkyGrayTint01,
+  textPrimaryColor,
 } from 'bpk-tokens/tokens/base.react.native';
-import { Text, StyleSheet } from 'react-native';
+import { Text } from 'react-native';
 import React, { type ElementProps } from 'react';
 import iconMappings from 'bpk-svgs/dist/font/iconMapping.json';
+import {
+  BpkDynamicStyleSheet,
+  useBpkDynamicStyleSheet,
+} from 'react-native-bpk-appearance';
 
-const styles = StyleSheet.create({
+const dynamicStyles = BpkDynamicStyleSheet.create({
   icon: {
     fontFamily: 'BpkIcon',
     fontSize: spacingLg,
     includeFontPadding: false,
-    color: colorSkyGrayTint01,
+    color: textPrimaryColor,
```

Comment:
As this is an icon and not text is `textPrimaryColor` an appropriate value to be using here?

Reply:
It is actually a `text` component :).I would say yes because It is expected to have the same colour as text in normal situations.

It is also the token iOS and Android use